### ISSUE TITLE
lightning-cli fundchannel - command not found

### ISSUE THREAD

**Issue by wiziwk** [shesek/spark-wallet#140]
lightning-cli fundchannel is not available in the bundle. Do I use the fundchannel_start and fundchannel_complete to fund a channel?

**Comment by shesek** [comment]
What do you mean by not available in the bundle?

You can create a new channel by opening the node details page (on the footer), clicking "Channels", then "Open Channel".

You can also create a channel using the RPC console within Spark. To do this, you need to enable expert mode (by clicking the version number in the footer), then open the "Console" page that'll be shown on the homepage. There, you should be able to issue `fundchannel` commands as you normally would.

**Comment by wiziwk** [comment]
exactly how I expect it to be but it isn't. If I issued "lightning-cli --help", it is returning the following parameters under ==channels== fundchannel_cancel, fundchannel_complete, fundchannel_start..." and it is missing fundchannel command.

If I use the browser to open a new channel, "Unknown command 'fundchannel'" is displayed bold red on the web page.

![Capture](https://user-images.githubusercontent.com/36327718/72668671-43fcc100-3a64-11ea-88d9-f739db6f8179.PNG)

Sorry I should have mentioned that I have installed the spark-wallet via docker pull.
